staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/redis/client.go

```diff
@@ -69,33 +69,16 @@ func (d *DataBaseConfig) AddData(data *common.DataModel) error {
 	// The key to construct the ordered set, here DeviceName is used as the key
 	klog.V(1).Infof("deviceName:%s", data.DeviceName)
 	// Check if the current ordered set exists
-	exists, err := RedisCli.Exists(ctx, data.DeviceName).Result()
+	deviceData := "TimeStamp: " + strconv.FormatInt(data.TimeStamp, 10) + " PropertyName: " + data.PropertyName + " data: " + data.Value
+	// Add data to ordered set. If the ordered set does not exist, it will be created.
+	_, err = RedisCli.ZAdd(ctx, data.DeviceName, &redis.Z{
```

Comment:
Does the key need a namespace? Do we need to add another recognizable prefix?

Reply:
Yes, the prefix here needs to be replaced by deviceID. I will submit another PR to modify the influx、redis、mysql 、tdengine uniformly.